### PR TITLE
docs(nms): Additional contribution guidance

### PR DESCRIPTION
**Pull request by andreilee**
Signed-off-by: Andrei Lee <andreilee@fb.com>

## Summary

Add more contribution guidance for NMS

## Test Plan

N/A

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: docs/readmes/contributing/contribute_conventions.md [hunk 1]
```diff
@@ -308,6 +308,8 @@ Where applicable, the [Google C++ Style Guide](https://google.github.io/stylegui
 
 #### General
 
+- The [Airbnb Javascript Style Guide](https://github.com/airbnb/javascript) is authoritative
+- See `magma/nms/README.md` for full details
```
Comment (themarwhal) [inline docs/readmes/contributing/contribute_conventions.md:312]: why don't we just put all the style doc in this page?

Comment (electronjoe) [reply docs/readmes/contributing/contribute_conventions.md:312]: Are you suggesting we copy the aribnb style content into here?

I suppose we may eventually want to diverge and would then want a copy - but this is a huge style guide.. And we likely cite e.g. Google Style elsewhere for C++?

**Review by electronjoe — APPROVED** [review]

File: docs/readmes/contributing/contribute_conventions.md (hunk 1 above)
Comment (themarwhal) [reply docs/readmes/contributing/contribute_conventions.md:312]: No I just meant what's inside `magma/nms/README.md` :)

File: nms/README.md [hunk 2]
```diff
@@ -120,6 +120,25 @@ Check the `magmalte` container logs.
 This likely corresponds to React errors.
 If this is the case, you should be able to see the relevant error logs through your web browser's developer tools.
 
+---
+## Contributing
+
+Please follow the guidance in this section when opening pull requests that touch NMS code.
+
+### Copyright Header
+All Javascript files should start with a Magma copyright notice.
```
Comment (hcgatewood) [inline nms/README.md:129]: Nit
```suggestion
All JavaScript files should start with a Magma copyright notice.
```

File: nms/README.md [hunk 3]
```diff
@@ -120,6 +120,25 @@ Check the `magmalte` container logs.
...
+### Style Guide
+As of October 2021, we are adopting the [Airbnb Javascript Style Guide](https://github.com/airbnb/javascript) for NMS code.
+
+### Verification
+Before submitting a pull request, make sure that both automated and manual verification is completed. Run Flow and Eslint checks for style issues. Run and update unit tests for all changes. Manual checks should also be used. Bring up a local development NMS, and work through the user flows that are impacted by your changes.
+
+### Pull Requests
+When submitting a pull request, include pictures of altered interfaces, and videos of altered user flows.
+
+Consult the `Testing` section below for additional details.
```
Comment (hcgatewood) [inline nms/README.md:140]: Nit
```suggestion
Consult the [Testing](#testing) section below for additional details.
```

File: docs/readmes/contributing/contribute_conventions.md (hunk 1 above)
Comment (hcgatewood) [reply docs/readmes/contributing/contribute_conventions.md:312]: Agreed, let's pass by value here rather than reference

**Review by hcgatewood — APPROVED** [review]
👍👍

**Review by HannaFar — APPROVED** [review]

**Pull request closed without merging**